.golangci.yaml

```diff
@@ -1,26 +1,54 @@
-linters-settings:
-  custom:
-    kubeapilinter:
-      type: "module"
-      description: kubeapilinter is the Kube-API-Linter and lints Kube like APIs based on API conventions and best practices.
-      settings:
-        linters:
-          enable:
-          - "maxlength"
-          - "nobools"
-          - "nomaps"
-          - "statussubresource"
-        lintersConfig:
-          conditions:
-            isFirstField: Warn
-            useProtobuf: Ignore
-            usePatchStrategy: Ignore
+version: "2"
 linters:
-  disable-all: true
+  default: none
   enable:
     - kubeapilinter
+  settings:
+    custom:
+      kubeapilinter:
+        path: tools/_output/bin/kube-api-linter.so
+        description: kubeapilinter is the Kube-API-Linter and lints Kube like APIs based on API conventions and best practices.
+        settings:
+          linters:
+            enable:
+              - maxlength
+              - nobools
+              - nomaps
+              - statussubresource
+          lintersConfig:
```

Comment:
Should we add a configuration for the new `uniquemarkers` linter to prevent duplicate feature-gate markers on fields and types?